app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,88 @@
  *= require_tree .
  *= require_self
  */
+
+/*fonts*/
+@import url(https://fonts.googleapis.com/css?family=Poller+One|Ubuntu:700,400);
```

Comment:
It is recommended to pull this in in the html e.g.
`<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poller+One|Ubuntu:700,400">`